src/components/common/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -224,6 +220,14 @@ export default class NameTranslation extends Mixins(CommonMixin) {
     return this.draftTranslations.filter(x => x.action !== ActionTypes.REMOVED)
   }
 
+  /** Returns true when user can edit the name translations. */
+  get isEditNameTranslationButtonVisible (): boolean {
+    return (!this.hasNameTranslationChange &&
+    !this.isSummaryMode &&
+    !this.isLimitedConversionRestorationFiling &&
+    !this.isLimitedExtendRestorationFiling)
+  }
```

Comment:
What do you think about this style instead?
```
return (
 condition1 &&
 condition2 ...
)
```

Reply:
I was reluctant to mess with the logic.  Are you suggesting removing the `!` in front of each condition?  I just noticed that I should change the method's name and comment to make clear that the get method returns true when the button should be hidden -- the logic is correct but the wording is incorrect.

Reply:
The style, not the logic.

Reply:
If you hate the logic, you could do this:
```
if (a || b || c) return true
return false
```

Reply:
Ah, got it, thanks for the clarification.  I've changed both the format and the cleaned up the logic so it's not this and not that etc.  Let me know if that works for you.